src/System.Net.Mail/src/System.Net.Mail.csproj

```diff
@@ -6,6 +6,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <IsPartialFacadeAssembly Condition="'$(TargetGroup)'=='net463'">true</IsPartialFacadeAssembly>
     <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.7</NuGetTargetMoniker>
+    <DefineConstants Condition="'$(TargetGroup)' == ''">NETSTANDARD1_7</DefineConstants>
```

Comment:
Thanks @weshaggard. Makes sense.
In this case the condition has been removed in a later change.